pkg/mons/restore_quorum.go

```diff
@@ -278,13 +278,15 @@ func getMonDetails(goodMon string, monEndpoints []string) ([]string, string, str
 	return badMons, goodMonPublicIp, goodMonPort, nil
 }
 
-func promptToContinueOrCancel(answer string) (string, error) {
+func PromptToContinueOrCancel(inputSource, answer string) (string, error) {
```

Comment:
How about the expected answer as a parameter? Then you don't need an if statement to hard code the answer inside the method
```suggestion
func PromptToContinueOrCancel(inputSource, expectedAnswer, answer string) (string, error) {
```

Reply:
done